src/highlightjs-adapter.ts

```diff
@@ -1,36 +1,36 @@
-// from https://github.com/cdnjs/SRIs/blob/master/highlight.js/9.18.3.json
-import * as sri from './highlightjs-sri-9.18.3.json'
+import * as vscode from 'vscode'
+import * as path from 'path'
+import { WebviewPanel } from "vscode";
```

Comment:
We don't need this, right?

```suggestion
```

Reply:
You are correct, I'm using `vscode.WebviewPanel` so `import * as vscode from 'vscode'` is sufficient.